serverless/src/exportSearch/handler.js

```diff
@@ -72,20 +115,37 @@ const exportSearch = async (event, context) => {
           }
         },
         headers: requestHeaders
-      })
+      }
+
+      // sleep 2 seconds before each request except the first
+      // so as not to over-load cmr-graphql
```

Comment:
I don't think we have to be worried about overloading cmr-graphql, its runs on lambda so it will use as many lambdas as it needs to meet demand, I think all this logic can be removed.

Were you seeing errors coming back from these requests?

Reply:
removed sleep with https://github.com/nasa/earthdata-search/pull/1630/commits/90191ef3d6ed3344229ac264ded2adc3bc53e974